src/main/scala/org/constellation/domain/consensus/ConsensusService.scala

```diff
@@ -3,19 +3,20 @@ package org.constellation.domain.consensus
 import cats.effect.concurrent.Semaphore
 import cats.effect.{Concurrent, ContextShift, IO, Sync}
 import cats.implicits._
-import io.chrisdavenport.log4cats.Logger
 import org.constellation.ConstellationExecutionContext
 import org.constellation.primitives.Schema.CheckpointCache
 import org.constellation.primitives.concurrency.SingleLock
 import ConsensusStatus.ConsensusStatus
+import io.chrisdavenport.log4cats.slf4j.Slf4jLogger
 import org.constellation.storage.algebra.{Lookup, MerkleStorageAlgebra}
 import org.constellation.storage.{ConcurrentStorageService, PendingMemPool, StorageService}
 
-abstract class ConsensusService[F[_]: Concurrent: Logger, A <: ConsensusObject]
-    extends MerkleStorageAlgebra[F, String, A] {
+abstract class ConsensusService[F[_]: Concurrent, A <: ConsensusObject] extends MerkleStorageAlgebra[F, String, A] {
 
   def metricRecordPrefix: Option[String]
 
+  private val logger = Slf4jLogger.getLogger[F]
```

Comment:
can't we just mix-in `with StrictLogging`?

```scala
/**
 * Defines `logger` as a value initialized with an underlying `org.slf4j.Logger`
 * named according to the class into which this trait is mixed.
 */
trait StrictLogging {

  protected val logger: Logger =
    Logger(LoggerFactory.getLogger(getClass.getName))
}
```

Reply:
We're using a different `Logger` than from `StrictLogging`. The logger from cats is instantiated under `F` type class and `logger.debug` is like a `Sync[F].delay { logger.debug .. }` from StrictLogging